benchmark/synthetic/models/lstm_text_generation_wikitext2.py

```diff
@@ -0,0 +1,190 @@
+'''
```

Comment:
Would recommend using a helper file to for benchmarking both these datasets - as theres a lot of common code between the two. Or you could create an array with the file names & iterate over that array in one benchmarking script to prevent redundancy in code.